services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/liitteet/AttachmentsController.kt

```diff
@@ -0,0 +1,113 @@
+package fi.hel.haitaton.hanke.liitteet
+
+import fi.hel.haitaton.hanke.*
+import fi.hel.haitaton.hanke.permissions.PermissionCode
+import fi.hel.haitaton.hanke.permissions.PermissionService
+import java.util.*
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.multipart.MultipartFile
+
+@RestController
+@RequestMapping("/liitteet")
+class AttachmentsController(
+    @Autowired private val attachmentService: AttachmentService,
+    @Autowired private val hankeService: HankeService,
+    @Autowired private val permissionService: PermissionService,
+) {
+
+    @GetMapping("/hanke/{hankeTunnus}")
+    fun getAttachments(@PathVariable hankeTunnus: String): List<HankeAttachment> {
+        val currentUserId = currentUserId()
+        val hankeId =
+            hankeService.getHankeId(hankeTunnus) ?: throw HankeNotFoundException(hankeTunnus)
+
+        if (!permissionService.hasPermission(hankeId, currentUserId, PermissionCode.VIEW)) {
+            throw HankeNotFoundException(hankeTunnus)
+        }
+
+        val hankeAttachments = attachmentService.getHankeAttachments(hankeTunnus)
+        return hankeAttachments
+    }
+
+    @PostMapping("/hanke/{hankeTunnus}")
```

Comment:
I think these would be better as `/hanke/{hankeTunnus}/liitteet` since attachments are a part of hanke, not the other way around.

Reply:
The hanke related endpoints have been moved.

Reply:
I don't think there was any need to move the controller methods, changing the mappings would've been enough.

Reply:
I brought the endpoints back to attachmentcontroller. Fyi: I renamed the package from liitteet to attachment.